src/pep_data/test/t_conf_windows.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
```

Comment:
Why do you need this file? it has the same code as the adjacent t_project_conf.py. script, if you do pull request erase it from git history

Reply:
its not exactly the same.
you gave me this file in slack few weeks ago.
do you want me to delete that?